background/get-userscripts.js

```diff
@@ -32,12 +32,11 @@ async function getContentScriptInfo(url) {
     addonsWithUserscripts: [],
     userstyleUrls: [],
     themes: [],
+    allAddons: [],
```

Comment:
What is this for?

Reply:
"get-userscripts.js" needs the scripts that should be run for ALL addons. See where allAddons is used in cs.js and then in the other module.js file.